lib/vue/src/components/resourceList/CResourceList.vue

```diff
@@ -129,11 +129,21 @@
       v-if="showFooter"
       #footer
     >
-      <div
-        v-if="!hideTotal"
-        class="text-nowrap"
-      >
-        {{ getPagination }}
+      <div class="d-flex gap-1 align-items-center">
```

Comment:
It should wrap like recordList, styles should be similar. User shouldn't see a difference